consensus/hotstuff/verification/single_verifier.go

```diff
@@ -65,7 +64,6 @@ func (s *SingleVerifier) VerifyQC(voterIDs []flow.Identifier, sigData []byte, bl
 	if len(signers) < len(voterIDs) { // check we have valid consensus member Identities for all signers
```

Comment:
I would find the following cleaner:
```suggestion
	if len(signers) != len(voterIDs) { // check we have valid consensus member Identities for all signers
```

Because, we need _exactly_ `len(voterIDs)` identities. Otherwise, I have to think through the case `len(signers) > len(voterIDs)`, or rather, why this case can't happen.

Reply:
I can update it to `!=`.
In line 60, ```signers, err := s.committee.Identities(block.BlockID, filter.HasNodeID(voterIDs...))``` means that the set of signers is included in the set of voters, i.e `len(signers) <= len(voterIDs)`. I guess that's why `<` is enough.

Reply:
I think `len(signers) > len(voterIDs)` may happen if `s.committee` contains redundant entities (same nodeID) but different pointers. Do you think that's possible?

Reply:
updated in https://github.com/onflow/flow-go/pull/445